subcommittees/mentoring/mission-objectives.md

```diff
@@ -0,0 +1,48 @@
+# Mission
+
+The Mentoring Subcommittee was established in 2015 to develop and maintain a mentorship program to support instructors as they progress through training, teaching, curriculum development, and other community-related activities. We help promote community-building and networking by providing (virtual) spaces where instructors from all over the world can share teaching success stories and discuss strategies for overcoming challenges. 
```

Comment:
I was trying not to duplicate content, but it seemed important to me to have it both on the README (which I've written to be more "user facing") and in a more formal internal document (which this is).  

Maybe the answer is to put the current README content/format on the website + recombine the README/mission-objectives.md content into the README.